src/D2L.Security.OAuth2/IsExternalInit.net4x.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// From: https://github.com/dotnet/runtime/commit/6072e4d3a7a2a1493f514cdf4be75a3d56580e84#diff-afc57cc6e9c659ebf491bc78d06cdb5b13d5a049122d8a41f744a515f493663b
+
+using System.ComponentModel;
+
+namespace System.Runtime.CompilerServices {

```

Comment:
private polyfill, only for .NET Framework (see path -- `.net4x.cs` and the magic we do in csproj for multi-platform.)

Reply:
This is for `init;` in the `OAuth2Configuration` file -- I could drop it if it rankles.